code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -137,11 +137,12 @@
 	return
 
 /mob/living/simple_animal/bot/mulebot/screwdriver_act(mob/living/user, obj/item/I)
-	if(open)
-		on = FALSE
-	update_controls()
-	update_icon()
-	return TRUE
+	. = ..()
+	if(.)
```

Comment:
```suggestion
	if(.)
```
if the contents of this `if` are only being run if true, then guard clause it with an early return if false

Reply:
we prefer early returns always

Reply:
commited